lib/modules/datasource/packagist/index.ts

```diff
@@ -133,7 +133,10 @@ export class PackagistDatasource extends Datasource {
     registryUrl: string,
     registryMeta: RegistryMeta
   ): string | null {
+    const registryHost = parseUrl(registryUrl)?.origin;
+
```

Comment:
```suggestion

    if (is.nullOrUndefined(registryHost)) {
      return null;
    }
```
Is doing the same and is IMO more readable.

Reply:
Actually, I don't like the fact `parseUrl()` returns nullable result. There are many places we're sure URLs are valid, but still we have to check its result for null.